test/e2e/error.e2e.js

```diff
@@ -0,0 +1,74 @@
+import { chTest } from './helper';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+describe('ContactHub', () => {
+  describe('errors', async () => {
+    it('should return an unauthorized error (401)', async () => {
+      const fakeCh = chTest({ token: 'fake-token' });
+      try {
+        await fakeCh.getCustomers();
+      } catch (err) {
+        expect(err.status).toBe(401);
+        expect(err.message).toBe('The client is not authorized to access the API');
+      }
+    });
+
+    it('should return a workspace not found error (403)', async () => {
+      try {
+        await chTest({ workspaceId: 'fake-workspace-id' }).getCustomers();
+      } catch (err) {
+        expect(err.status).toBe(403);
+        expect(err.message).toBe('workspace not found: fake-workspace-id');
+      }
+    });
+
+    it('should return a node not found error (404)', async () => {
+      try {
+        await chTest({ nodeId: 'fake-node-id' }).getCustomers();
+      } catch (err) {
+        expect(err.status).toBe(404);
+        expect(err.message).toBe('node not foundfake-node-id');
+      }
+    });
+
+
+    it('should return a duplicated customer error (409)', async () => {
+      let customer;
+      const customerEmail = 'duplicated-email@fake-customer.com';
+      const customerData = { base: { contacts: { email: customerEmail } } };
+
+      const ch = chTest();
+
+      // try to add a customer for the first time to be sure it's present or fetch it from API
```

Comment:
If you create a new customer with a random email, then you call `addCustomer` twice you don't need the `query`, the first time it will always succeed, the second time it will always return 409.